src/components/App.jsx

```diff
@@ -34,7 +38,27 @@ const KeyboardAwareSidebar = ({ children }) => {
 
 const App = props => {
   const { showBottomNav, settingsCollection } = props
+  const { t } = useI18n()
+  const client = useClient()
   const settings = getDefaultedSettingsFromCollection(settingsCollection)
+
+  const showAlert = t => {
+    Alerter.error(t('Error.fetch-error'), {
+      buttonText: t('General.reload'),
+      buttonAction: () => window.location.reload()
+    })
+  }
+
+  useEffect(() => {
+    const onError = e => {
+      if (!getActivatePouch() && hasFetchFailedError(e)) {
+        throttle(showAlert(t), 300)
+      }
+    }
+
+    client.setOnError(onError)
```

Comment:
weird to call setOnError(undefined), no? It'll assign this.onError = undefined in CozyClient side. I find it weird. You should not call setOnError if not needed?

Reply:
Here onError is defined. setOnError is not mandatory so if you want use it it's possible. Maybe are you talking about other things?

Reply:
No you're right. My bad, thanks.